lisa/tools/hyperv.py

```diff
@@ -156,6 +164,18 @@ def create_vm(
         if not is_ready:
             raise LisaException(f"VM {name} did not start")
 
+    def get_default_external_switch(self) -> Optional[VMSwitch]:
```

Comment:
Move it to another PR.

Reply:
https://github.com/microsoft/lisa/pull/3170